### PR TITLE
Reorganize README to move templates to end

### PR DESCRIPTION
**Pull request by willingc**
Explosive growth of cookiecutters is a nice problem to have. Here is a WIP PR for putting the templates and similar projects to the end of the README. An earlier section links to the templates

I added a section for cookiecutter maintained templates but I wasn't sure which were or were not.

Addresses #662 in part.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -148,7 +148,118 @@ Did someone say features?
 Available Cookiecutters
 -----------------------
 
-Here is a list of **cookiecutters** (aka Cookiecutter project templates) for you to use or fork.
+Making great cookies takes a lot of cookiecutters and contributors. We're so
+pleased that there are many Cookiecutter project templates to choose. We hope
+you find a cookiecutter that is just right for your needs.
```
Comment (hackebrot) [inline README.rst]: 👍

File: README.rst [hunk 2]
```diff
@@ -148,7 +148,118 @@ Did someone say features?
...
+`Common Lisp`_ | 
+`JS`_ | 
+`Kotlin`_ | 
+`LaTeX/XeTeX`_ | 
+`PHP`_ | 
+`Berkshelf-Vagrant`_ | 
+`HTML`_ | 
+`Scala`_ | 
+`6502 Assembly`_ | 
+`Similar Projects`_
```
Comment (hackebrot) [inline README.rst]: "Similar Projects" should have its own section, I think.

File: README.rst [hunk 3]
```diff
@@ -148,7 +148,118 @@ Did someone say features?
 Available Cookiecutters
 -----------------------
 
-Here is a list of **cookiecutters** (aka Cookiecutter project templates) for you to use or fork.
+Making great cookies takes a lot of cookiecutters and contributors. We're so
+pleased that there are many Cookiecutter project templates to choose. We hope
```
Comment (michaeljoseph) [inline README.rst]: to choose **from**?

**Comment by michaeljoseph** [comment]
@willingc I think the relevant team cookiecutters should be:
https://github.com/audreyr/cookiecutter-pypackage
https://github.com/pydanny/cookiecutter-django
https://github.com/pytest-dev/cookiecutter-pytest-plugin

**Comment by hackebrot** [comment]
Great! I love it. 👍 

I'd be happy to merge this if there are no objections. /cc @audreyr @pydanny @michaeljoseph

**Comment by audreyr** [comment]
Nice work @willingc.

I'm about to work on reviewing template PRs now, which touch a lot of the same lines...to prevent this PR from getting stale I will go ahead and merge this first.

**Pull request merged**